suite-native/atoms/src/StepsProgressBar.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { Box } from './Box';
+
+type StepsProgressBarProps = {
+    steps: number[];
```

Comment:
use just number, passing array as prop instead of number primitive will probably cause unnecessary rerenders

you can easily generate array from number using `A.makeWithIndex(numberOfSteps, index => index + 1)`

Reply:
[8b0ee66](https://github.com/trezor/trezor-suite/pull/6591/commits/8b0ee665c807585c82c820c3a6dc95b0806c88c0)